source/process/sg_rest_markup.rst

```diff
@@ -83,31 +93,24 @@ Create numbered lists and procedure steps using arabic numerals for the top-leve
     
   3. This is item three.
   
-Links to external web pages
----------------------------
+External Links
+---------------
 
-The quickest and easiest way to create a link in reStructuredText is to use the URL without any extra markup. For example:
+URLs will automatically be rendered as links in Sphinx; however, where possible, it is preferred that hyperlinks are created within the text of a sentence. Hyperlinks within a sentence can be created using the following formatting:
 
-.. code-block:: none
-  
-  https://www.mattermost.org/manifesto/
-
-On output to HTML, the link looks like this: https://www.mattermost.org/manifesto/.
-
-You can also create a link that has link text. For example:
+``Link display text <URL-of-website>`_``, for example:
 
 .. code-block:: none
   
   `Mattermost Manifesto <https://www.mattermost.org/manifesto/>`_
 
-On output to HTML, the link looks like this: `Mattermost Manifesto <https://www.mattermost.org/manifesto/>`_
-
-Pay close attention to the syntax. The link starts with a backquote character, followed by the text that you want to see in the document, a final space, the < character, the URL, the > character, a closing backquote, and finally an underscore.
+The link renders as: `Mattermost Manifesto <https://www.mattermost.org/manifesto/>`_
 
 .. _relative-links-in-rst:
 
-Links to targets within the Mattermost docs
--------------------------------------------
+
+Internal Links to Mattermost Docs
+----------------------------------
 
 The Sphinx processor extends reStructuredText to implement references to locations within a documentation set. The extensions are called `roles`, and the two roles that are relevant in Mattermost documentation are the ``:doc:`` role and the ``:ref:`` role.
```

Comment:
Should we update guideline to include the formatting for `roles` ? It's not clear if we should be using `roles`, "roles", or something else